src/reducers/dashboard/index.js

```diff
@@ -51,9 +52,15 @@ export function fetchDashboard(dispatch) {
 }
 
 function dashboardRequest(dispatch) {
-  return fetch(`${apiPath}/sessions`)
+  return fetch(`${apiPath}/sessions`, {
```

Comment:
This request also forgot cookies. I am not sure if this is by design or not. In my implementation the `sessions` endpoint requires authentication.